### ISSUE TITLE
Improvements to IgA protein complex terms

### ISSUE THREAD

**Issue by addiehl** [PROconsortium/PRoteinOntology#187]
I am proposing to change the IgA terms to allow for representation of the secretory and membrane -bound forms of the IgA1 and IgA2 isotypes in the appropriate protein complexes.  This issue includes new terms for the secretory and membrane-bound forms of the IgA1 and IgA2 heavy chains, and changes to the IgA1 and IgA2 protein complex terms including improved definitions and changes in the logical axioms.  Changed lines in the term stanzas below are marked with '***'.

I noted two inconsistencies that should probably be rectified as part of this request:
1) in the original IgA and IgD terms taxon restrictions were indicated using equivalence axioms that appear in the obo representation as (for example)
[Term]
id: PR:000050179
name: IgA1 heavy chain protein (human)
def: "An IgA1 heavy chain protein in human." [PRO:DAN]
comment: Category=organism-gene.
synonym: "hIgHA1" EXACT PRO-short-label [PRO:DAN]
intersection_of: PR:000050177 ! IgA1 heavy chain protein
intersection_of: only_in_taxon NCBITaxon:9606 ! Homo sapiens

whereas in the IgG terms that were recently added have taxon restriction indicated using subclass axioms that appear the the obo representation as (for example)
[Term]
id: PR:000050297
name: IgG1 heavy chain protein, secretory isoform (human)
def: "An IgG1 heavy chain protein (human) that is a translation product of some mRNA polyadenylated at a site 5' to the membrane exons, thus preventing their incorporation the final spliced mRNA." [PRO:DAN, PRO:ADD, PMID:9185563, PMID:17496345]
comment: Category=organism-seqgroup. Requested by=HIPC. Requested by=CL.
synonym: "hIgHG1/iso:secretory" EXACT PRO-short-label [PRO:ADD]
synonym: "immunoglobulin G1 heavy chain, secretory isoform (human)" EXACT [PRO:ADD]
synonym: "immunoglobulin gamma 1 heavy chain, secretory isoform (human)" EXACT [PRO:ADD]
synonym: "immunoglobulin heavy chain gamma 1, secretory isoform (human)" EXACT [PRO:ADD]
is_a: PR:000050291 ! IgG1 heavy chain protein (human)
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

I have adopted the second form (use of subclass axioms) for the terms that are part of this submission.

2) I have used GOC:add to designate my ontology contributions across the OBO Foundry for many years, which is linked to my ORCID URI through the file at
https://github.com/geneontology/go-site/blob/master/metadata/users.yaml

I would prefer my contributions to PRO use GOC:add, rather than PRO:ADD for this reason, if possible.  Note that GOC:add is used in some of my PRO contributions already, and PRO:ADD in others.

Thanks,
Alex

**Improvements to IgA terms follow**

**Existing terms (changes possibly needed in taxon restrictions in some cases)**
[Term]
id: PR:000050177
name: IgA1 heavy chain protein
def: "An IgA heavy chain protein in which the constant region is encoded by the IgA1 gene segment." [PRO:DAN]
comment: Category=gene.
synonym: "IgHA1" EXACT PRO-short-label [PRO:DAN]
is_a: PR:000050176 ! IgA heavy chain protein

[Term]
id: PR:000050178
name: IgA2 heavy chain protein
def: "An IgA heavy chain protein in which the constant region is encoded by the IgA2 gene segment." [PRO:DAN]
comment: Category=gene.
synonym: "IgHA2" EXACT PRO-short-label [PRO:DAN]
is_a: PR:000050176 ! IgA heavy chain protein

[Term]
id: PR:000050179
name: IgA1 heavy chain protein (human)
def: "An IgA1 heavy chain protein in human." [PRO:DAN]
comment: Category=organism-gene.
synonym: "hIgHA1" EXACT PRO-short-label [PRO:DAN]
intersection_of: PR:000050177 ! IgA1 heavy chain protein
intersection_of: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050180
name: IgA2 heavy chain protein (human)
def: "An IgA1 heavy chain protein in human." [PRO:DAN]
comment: Category=organism-gene.
synonym: "hIgHA2" EXACT PRO-short-label [PRO:DAN]
intersection_of: PR:000050178 ! IgA2 heavy chain protein
intersection_of: only_in_taxon NCBITaxon:9606 ! Homo sapiens

**New terms**
[Term]
id: PR:NewTerm01
name: IgA1 heavy chain protein, secretory isoform
def: "An IgA1 heavy chain protein in that is a translation product of an IgA1 mRNA polyadenylated at a site 5' to the membrane exons, thus preventing their incorporation the final spliced mRNA." [PRO:DAN, GOC:add]
comment: Category=organism-gene.
synonym: "IgHA1/iso:secretory" EXACT PRO-short-label [PRO:DAN]
is_a: PR:000050177 ! IgA1 heavy chain protein

[Term]
id: PR:NewTerm02
name: IgA1 heavy chain protein, membrane-bound isoform
def: "An IgA1 heavy chain protein that is a translation product of an IgA1 mRNA polyadenylated at a site 3' to the membrane exons." [PRO:DAN, GOC:add]
comment: Category=organism-gene.
synonym: "IgHA1/iso:membrane-bound" EXACT PRO-short-label [PRO:DAN]
is_a: PR:000050177 ! IgA1 heavy chain protein

[Term]
id: PR:NewTerm03
name: IgA2 heavy chain protein, secretory isoform
def: "An IgA2 heavy chain protein in that is a translation product of an IgA2 mRNA polyadenylated at a site 5' to the membrane exons, thus preventing their incorporation the final spliced mRNA." [PRO:DAN, GOC:add]
comment: Category=organism-gene.
synonym: "IgHA2/iso:secretory" EXACT PRO-short-label [PRO:DAN]
is_a: PR:000050178 ! IgA2 heavy chain protein

[Term]
id: PR:NewTerm04
name: IgA2 heavy chain protein, membrane-bound isoform
def: "An IgA2 heavy chain protein that is a translation product of an IgA2 mRNA polyadenylated at a site 3' to the membrane exons." [PRO:DAN, GOC:add]
comment: Category=organism-gene.
synonym: "IgHA2/iso:membrane-bound" EXACT PRO-short-label [PRO:DAN]
is_a: PR:000050178 ! IgA2 heavy chain protein

[Term]
id: PR:NewTerm05
name: IgA1 heavy chain protein, secretory isoform (human)
def: "An IgA1 heavy chain protein in human that is a translation product of an IgA1 mRNA polyadenylated at a site 5' to the membrane exons, thus preventing their incorporation the final spliced mRNA." [PRO:DAN, GOC:add]
comment: Category=organism-gene.
synonym: "hIgHA1/iso:secretory" EXACT PRO-short-label [PRO:DAN]
is_a: PR:NewTerm01 ! IgA1 heavy chain protein, secretory isoform
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:NewTerm06
name: IgA1 heavy chain protein, membrane bound isoform (human)
def: "An IgA1 heavy chain protein in human that is a translation product of an IgA1 mRNA polyadenylated at a site 3' to the membrane exons." [PRO:DAN, GOC:add]
comment: Category=organism-gene.
synonym: "hIgHA1/iso:membrane-bound" EXACT PRO-short-label [PRO:DAN]
is_a: PR:NewTerm02 ! IgA1 heavy chain protein, membrane-bound isoform
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:NewTerm07
name: IgA2 heavy chain protein, secretory isoform (human)
def: "An IgA2 heavy chain protein in human that is a translation product of an IgA2 mRNA polyadenylated at a site 5' to the membrane exons, thus preventing their incorporation the final spliced mRNA." [PRO:DAN, GOC:add]
comment: Category=organism-gene.
synonym: "hIgHA2/iso:secretory" EXACT PRO-short-label [PRO:DAN]
is_a: PR:NewTerm03 ! IgA2 heavy chain protein, secretory isoform
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:NewTerm08
name: IgA2 heavy chain protein, membrane bound isoform (human)
def: "An IgA2 heavy chain protein in human that is a translation product of an IgA2 mRNA polyadenylated at a site 3' to the membrane exons." [PRO:DAN, GOC:add]
comment: Category=organism-gene.
synonym: "hIgHA2/iso:membrane-bound" EXACT PRO-short-label [PRO:DAN]
is_a: PR:NewTerm04 ! IgA2 heavy chain protein, membrane-bound isoform
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

**Modified terms**
[Term]
id: PR:000050159
name: IgA1 immunoglobulin complex, circulating (human)
*** def: "A protein complex that in monomeric form is composed of two identical immunoglobulin heavy chains of the IgA1 isotype of the secretory isoform and two identical immunoglobulin light chains (kappa or lambda), held together by disulfide bonds, sometimes complexed with J chain or J chain and secretory component, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
comment: Category=organism-complex. Requested by=HIPC. Requested by=CL.
synonym: "hIgA1/circulating" EXACT PRO-short-label [PRO:DAN]
synonym: "IgA antibody (human)" BROAD [GOC:add]
synonym: "IgA1 antibody (human)" EXACT [GOC:add]
is_a: GO:0071746 ! IgA immunoglobulin complex, circulating
*** relationship: has_component PR:NewTerm05 ! IgA1 heavy chain protein, secretory isoform (human)
relationship: has_component PR:000050183 ! immunoglobulin light chain protein (human)
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050160
name: IgA2 immunoglobulin complex, circulating (human)
*** def: "A protein complex composed of two identical immunoglobulin heavy chains of the IgA2 isotype of the secretory isoform and two identical immunoglobulin light chains (kappa or lambda), held together by disulfide bonds, sometimes complexed with J chain or J chain and secretory component, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
comment: Category=organism-complex. Requested by=HIPC. Requested by=CL. Note: Human circulating IgA2 is only found as a monomeric form.
synonym: "hIgA2/circulating" EXACT PRO-short-label [PRO:DAN]
synonym: "IgA antibody (human)" BROAD [GOC:add]
synonym: "IgA2 antibody (human)" EXACT [GOC:add]
synonym: "monomeric IgA antibody (human)" BROAD [GOC:add]
synonym: "monomeric IgA2 antibody (human)" EXACT [GOC:add]
synonym: "monomeric IgA2 immunoglobulin complex (human)" EXACT [GOC:add]
is_a: GO:0071746 ! IgA immunoglobulin complex, circulating
*** relationship: has_component PR:NewTerm07 {cardinality="2"} ! IgA2 heavy chain protein, secretory isoform (human)
relationship: has_component PR:000050183 {cardinality="2"} ! immunoglobulin light chain protein (human)
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050161
name: IgA1 B cell receptor complex (human)
*** def: "An IgA1 immunoglobulin complex that is present in the plasma membrane of B cells and is composed of two identical immunoglobulin heavy chains of the membrane-bound isoform of the IgA1 isotype, two identical immunoglobulin light chains (kappa or lambda), and a signaling subunit, a heterodimer of the Ig-alpha and Ig-beta proteins." [GOC:add, ISBN:0781765196, PMID:16362985]
comment: Category=organism-complex. Requested by=HIPC. Requested by=CL.
synonym: "hIgA1/B-cell-receptor" EXACT PRO-short-label [PRO:DAN]
synonym: "membrane-bound IgA (human)" BROAD [GOC:add]
synonym: "membrane-bound IgA1 (human)" EXACT [GOC:add]
synonym: "surface IgA (human)" BROAD [GOC:add]
synonym: "surface IgA1 (human)" EXACT [GOC:add]
is_a: GO:0071747 ! IgA B cell receptor complex
relationship: part_of GO:0005886 ! plasma membrane
relationship: part_of CL:0000236 ! B cell
*** relationship: has_component PR:NewTerm06 {cardinality="2"} ! IgA1 heavy chain protein, membrane bound isoform (human)
relationship: has_component PR:000050183 {cardinality="2"} ! immunoglobulin light chain protein (human)
relationship: has_component PR:P11912 {cardinality="1"} ! B-cell antigen receptor complex-associated protein alpha chain (human)
relationship: has_component PR:P40259 {cardinality="1"} ! B-cell antigen receptor complex-associated protein beta chain (human)
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050162
name: IgA2 B cell receptor complex (human)
*** def: "An IgA2 immunoglobulin complex that is present in the plasma membrane of B cells and is composed of two identical immunoglobulin heavy chains of the membrane-bound isoform of the IgA2 isotype, two identical immunoglobulin light chains (kappa or lambda), and a signaling subunit, a heterodimer of the Ig-alpha and Ig-beta proteins." [GOC:add, ISBN:0781765196, PMID:16362985]
comment: Category=organism-complex. Requested by=HIPC. Requested by=CL.
synonym: "hIgA2/B-cell-receptor" EXACT PRO-short-label [PRO:DAN]
synonym: "membrane-bound IgA (human)" BROAD [GOC:add]
synonym: "membrane-bound IgA2 (human)" EXACT [GOC:add]
synonym: "surface IgA (human)" BROAD [GOC:add]
synonym: "surface IgA2 (human)" EXACT [GOC:add]
is_a: GO:0071747 ! IgA B cell receptor complex
relationship: part_of GO:0005886 ! plasma membrane
relationship: part_of CL:0000236 ! B cell
*** relationship: has_component PR:NewTerm08 {cardinality="2"} ! IgA2 heavy chain protein, membrane bound isoform (human)
relationship: has_component PR:000050183 {cardinality="2"} ! immunoglobulin light chain protein (human)
relationship: has_component PR:P11912 {cardinality="1"} ! B-cell antigen receptor complex-associated protein alpha chain (human)
relationship: has_component PR:P40259 {cardinality="1"} ! B-cell antigen receptor complex-associated protein beta chain (human)
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050163
name: monomeric IgA1 immunoglobulin complex (human)
*** def: "A protein complex composed of two identical immunoglobulin heavy chains of the secretory isoform of the IgA1 isotype and two identical immunoglobulin light chains (kappa or lambda), held together by disulfide bonds, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
comment: Category=organism-complex. Requested by=HIPC. Requested by=CL.
synonym: "h(IgA1)monomeric" EXACT PRO-short-label [PRO:DAN]
synonym: "monomeric IgA antibody (human)" BROAD [GOC:add]
synonym: "monomeric IgA1 antibody (human)" EXACT [GOC:add]
*** is_a: PR:000050159 ! IgA1 immunoglobulin complex, circulating (human)
relationship: has_component PR:NewTerm05 {cardinality="2"} ! IgA1 heavy chain protein, secretory isoform (human)
relationship: has_component PR:000050183 {cardinality="2"} ! immunoglobulin light chain protein (human)
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050165
name: polymeric IgA1 immunoglobulin complex (human)
*** def: "A protein complex composed of two, three, or four monomeric IgA1 immunoglobulin complexes linked through both direct disulfide bonds and through disulfide binded monomers of J chain acting as a bridge. Each IgA1 monomer consists of two identical immunoglobulin heavy chains of the secretory isoform of the IgA1 isotype and two identical immunoglobulin light chains (kappa or lambda), held together by disulfide bonds. Dimeric IgA1 is sometimes complexed additionally with secretory component, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
comment: Category=organism-complex. Requested by=HIPC. Requested by=CL. Dimeric IgA1 is by far the most common form of polymeric IgA1. In human only the IgA1 isotype is capable of a polymeric forms.
synonym: "h(IgA1)polymeric" EXACT PRO-short-label [PRO:DAN]
synonym: "pIgA antibody (human)" BROAD [GOC:add]
synonym: "pIgA1 antibody (human)" EXACT [GOC:add]
synonym: "polymeric IgA antibody (human)" BROAD [GOC:add]
synonym: "polymeric IgA1 antibody (human)" EXACT [GOC:add]
intersection_of: PR:000050159 ! IgA1 immunoglobulin complex, circulating (human)
intersection_of: has_component PR:000050163 {minCardinality=2} ! monomeric IgA1 immunoglobulin complex (human)
intersection_of: has_component PR:000050163 {maxCardinality=4} ! monomeric IgA1 immunoglobulin complex (human)
intersection_of: has_component PR:P01591 {cardinality=1} ! immunoglobulin J chain (human)
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050166
name: dimeric IgA1 immunoglobulin complex (human)
*** def: "A protein complex composed of two monomeric IgA immunoglobulin complexes linked through both direct disulfide bonds and through a disulfide binded monomer of J chain acting as a bridge. Each IgA1 monomer consists of two identical immunoglobulin heavy chains of the secretory isoform of the IgA isotype and two identical immunoglobulin light chains (kappa or lambda), held together by disulfide bonds. Dimeric IgA1 is sometimes complexed additionally with secretory component, and present in the extracellular space, in mucosal areas or other tissues, or circulating in the blood or lymph." [GOC:add, ISBN:0781765196, PMID:16362985]
comment: Category=organism-complex. Requested by=HIPC. Requested by=CL. Dimeric IgA1 is by far the most common form of polymeric IgA1. In human only the IgA1 isotype is capable of a dimeric form.
synonym: "h(IgA1)dimeric" EXACT PRO-short-label [PRO:DAN]
synonym: "dimeric IgA antibody (human)" BROAD [GOC:add]
synonym: "dimeric IgA1 antibody (human)" EXACT [GOC:add]
intersection_of: PR:000050165 ! polymeric IgA1 immunoglobulin complex (human)
intersection_of: has_component PR:000050163 {cardinality=2} ! monomeric IgA1 immunoglobulin complex (human)
intersection_of: has_component PR:P01591 {cardinality=1} ! immunoglobulin J chain (human)
relationship: only_in_taxon NCBITaxon:9606 ! Homo sapiens

**Comment by nataled** [comment]
Hi Alex, sorry for the delay in addressing this. A few quick notes on the 'inconsistencies' you note:

1) The first inconsistency arises from the manner of how the parent terms were used/created. In the first example, there is a species-neutral parent term, so the species-specific child term is simply the logical intersection of that parent and the taxon restriction. In the second example, the parent term is already species-specific, so the child uses an is_a relation (that is, the taxon 'restraint' isn't a restraint at all; it's mostly there for information purposes). Thus, if you request a species-neutral term (or one already exists) that can be used as a parent term, I use that.

2) With respect to attribution, the GOC:add was used whenever you requested a term without making it. PRO:ADD was used for those cases where you created the term. What I'll do is add (no pun intended!) GOC:add to the PRO:ADD cases.

I'll get back to you with further questions, if any.

**Comment by nataled** [comment]
Right on schedule, my first question: Other than the removal of synonyms and the Requested by tag (neither of which I will do), what changes are needed for the 'Existing terms'? They appear to be as they are already.

**Comment by addiehl** [comment]
I only stripped out the synonyms and requested by tags to make the GitHub issue shorter.  I didn't intend for you to change anything about the existing terms.  I simply put them there for easy reference.  Sorry for the confusion.

**Comment by nataled** [comment]
Gotcha. Below are the new requested terms. For the modified terms, I accepted them as they are except for those where the new definition implied 'protein complex' as the parent. In these cases they were changed to the actual parent as given by the is_a line.

PR:NewTerm01 = PR:000050372
PR:NewTerm02 = PR:000050373
PR:NewTerm03 = PR:000050374
PR:NewTerm04 = PR:000050375
PR:NewTerm05 = PR:000050376
PR:NewTerm06 = PR:000050377
PR:NewTerm07 = PR:000050378
PR:NewTerm08 = PR:000050379


[Term]
id: PR:000050372
name: IgA1 heavy chain protein, secretory isoform
def: "An IgA1 heavy chain protein that is a translation product of some mRNA polyadenylated at a site 5' to the membrane exons, thus preventing their incorporation the final spliced mRNA." [PRO:DAN, PRO:ADD, GOC:add]
comment: Category=seqgroup. Requested by=HIPC. Requested by=CL.
synonym: "IgHA1/iso:secretory" EXACT PRO-short-label [PRO:DAN]
synonym: "immunoglobulin A1 heavy chain, secretory isoform" EXACT [PRO:DAN]
synonym: "immunoglobulin alpha 1 heavy chain, secretory isoform" EXACT [PRO:DAN]
synonym: "immunoglobulin heavy chain alpha 1, secretory isoform" EXACT [PRO:DAN]
is_a: PR:000050177 ! IgA1 heavy chain protein

[Term]
id: PR:000050373
name: IgA1 heavy chain protein, membrane-bound isoform
def: "An IgA1 heavy chain protein that is a translation product of some mRNA polyadenylated at a site 3' to the membrane exons." [PRO:DAN, PRO:ADD, GOC:add]
comment: Category=seqgroup. Requested by=HIPC. Requested by=CL.
synonym: "IgHA1/iso:membrane-bound" EXACT PRO-short-label [PRO:DAN]
synonym: "immunoglobulin A1 heavy chain, membrane-bound isoform" EXACT [PRO:DAN]
synonym: "immunoglobulin alpha 1 heavy chain, membrane-bound isoform" EXACT [PRO:DAN]
synonym: "immunoglobulin heavy chain alpha 1, membrane-bound isoform" EXACT [PRO:DAN]
is_a: PR:000050177 ! IgA1 heavy chain protein

[Term]
id: PR:000050374
name: IgA2 heavy chain protein, secretory isoform
def: "An IgA2 heavy chain protein that is a translation product of some mRNA polyadenylated at a site 5' to the membrane exons, thus preventing their incorporation the final spliced mRNA." [PRO:DAN, PRO:ADD, GOC:add]
comment: Category=seqgroup. Requested by=HIPC. Requested by=CL.
synonym: "IgHA2/iso:secretory" EXACT PRO-short-label [PRO:DAN]
synonym: "immunoglobulin A2 heavy chain, secretory isoform" EXACT [PRO:DAN]
synonym: "immunoglobulin alpha 2 heavy chain, secretory isoform" EXACT [PRO:DAN]
synonym: "immunoglobulin heavy chain alpha 2, secretory isoform" EXACT [PRO:DAN]
is_a: PR:000050178 ! IgA2 heavy chain protein

[Term]
id: PR:000050375
name: IgA2 heavy chain protein, membrane-bound isoform
def: "An IgA2 heavy chain protein that is a translation product of some mRNA polyadenylated at a site 3' to the membrane exons." [PRO:DAN, PRO:ADD, GOC:add]
comment: Category=seqgroup. Requested by=HIPC. Requested by=CL.
synonym: "IgHA2/iso:membrane-bound" EXACT PRO-short-label [PRO:DAN]
synonym: "immunoglobulin A2 heavy chain, membrane-bound isoform" EXACT [PRO:DAN]
synonym: "immunoglobulin alpha 2 heavy chain, membrane-bound isoform" EXACT [PRO:DAN]
synonym: "immunoglobulin heavy chain alpha 2, membrane-bound isoform" EXACT [PRO:DAN]
is_a: PR:000050178 ! IgA2 heavy chain protein

[Term]
id: PR:000050376
name: IgA1 heavy chain protein, secretory isoform (human)
def: "An IgA1 heavy chain protein in human that is a translation product of an IgA1 mRNA polyadenylated at a site 5' to the membrane exons, thus preventing their incorporation the final spliced mRNA." [PRO:DAN, PRO:ADD, GOC:add]
comment: Category=organism-seqgroup. Requested by=HIPC. Requested by=CL.
synonym: "hIgHA1/iso:secretory" EXACT PRO-short-label [PRO:DAN]
synonym: "immunoglobulin A1 heavy chain, secretory isoform (human)" EXACT [PRO:DAN]
synonym: "immunoglobulin alpha 1 heavy chain, secretory isoform (human)" EXACT [PRO:DAN]
synonym: "immunoglobulin heavy chain alpha 1, secretory isoform (human)" EXACT [PRO:DAN]
intersection_of: PR:000050372 ! IgA1 heavy chain protein, secretory isoform
intersection_of: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050377
name: IgA1 heavy chain protein, membrane bound isoform (human)
def: "An IgA1 heavy chain protein in human that is a translation product of an IgA1 mRNA polyadenylated at a site 3' to the membrane exons." [PRO:DAN, PRO:ADD, GOC:add]
comment: Category=organism-seqgroup. Requested by=HIPC. Requested by=CL.
synonym: "hIgHA1/iso:membrane-bound" EXACT PRO-short-label [PRO:DAN]
synonym: "immunoglobulin A1 heavy chain, membrane-bound isoform (human)" EXACT [PRO:DAN]
synonym: "immunoglobulin alpha 1 heavy chain, membrane-bound isoform (human)" EXACT [PRO:DAN]
synonym: "immunoglobulin heavy chain alpha 1, membrane-bound isoform (human)" EXACT [PRO:DAN]
intersection_of: PR:000050373 ! IgA1 heavy chain protein, membrane-bound isoform
intersection_of: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050378
name: IgA2 heavy chain protein, secretory isoform (human)
def: "An IgA2 heavy chain protein in human that is a translation product of an IgA2 mRNA polyadenylated at a site 5' to the membrane exons, thus preventing their incorporation the final spliced mRNA." [PRO:DAN, PRO:ADD, GOC:add]
comment: Category=organism-seqgroup. Requested by=HIPC. Requested by=CL.
synonym: "hIgHA2/iso:secretory" EXACT PRO-short-label [PRO:DAN]
synonym: "immunoglobulin A2 heavy chain, secretory isoform (human)" EXACT [PRO:DAN]
synonym: "immunoglobulin alpha 2 heavy chain, secretory isoform (human)" EXACT [PRO:DAN]
synonym: "immunoglobulin heavy chain alpha 2, secretory isoform (human)" EXACT [PRO:DAN]
intersection_of: PR:000050374 ! IgA2 heavy chain protein, secretory isoform
intersection_of: only_in_taxon NCBITaxon:9606 ! Homo sapiens

[Term]
id: PR:000050379
name: IgA2 heavy chain protein, membrane bound isoform (human)
def: "An IgA2 heavy chain protein in human that is a translation product of an IgA2 mRNA polyadenylated at a site 3' to the membrane exons." [PRO:DAN, PRO:ADD, GOC:add]
comment: Category=organism-seqgroup. Requested by=HIPC. Requested by=CL.
synonym: "hIgHA2/iso:membrane-bound" EXACT PRO-short-label [PRO:DAN]
synonym: "immunoglobulin A2 heavy chain, membrane-bound isoform (human)" EXACT [PRO:DAN]
synonym: "immunoglobulin alpha 2 heavy chain, membrane-bound isoform (human)" EXACT [PRO:DAN]
synonym: "immunoglobulin heavy chain alpha 2, membrane-bound isoform (human)" EXACT [PRO:DAN]
intersection_of: PR:000050375 ! IgA2 heavy chain protein, membrane-bound isoform
intersection_of: only_in_taxon NCBITaxon:9606 ! Homo sapiens
